app/routes/organization/billing.js

```diff
@@ -1,7 +1,30 @@
 import TravisRoute from 'travis/routes/basic';
+import { inject as service } from '@ember/service';
+import { hash } from 'rsvp';
 
 export default TravisRoute.extend({
+
+  stripe: service(),
+
+  beforeModel() {
+    return this.stripe.load();
+  },
+
+  newSubscription() {
+    const billingInfo = this.store.createRecord('billing-info');
+    const plan = this.store.createRecord('plan');
+    const creditCardInfo = this.store.createRecord('credit-card-info');
+    return this.store.createRecord('subscription', {
+      billingInfo,
+      creditCardInfo,
+      plan
+    });
+  },
+
   model() {
-    return this.modelFor('organization');
+    return hash({
+      account: this.modelFor('organization'),
+      newSubscription: this.newSubscription()
+    });
```

Comment:
I see there is some shared logic with `account.billing` route. Would it make sense to extract it to a Mixin, to make the code DRYer?